charts/agent-control/Chart.yaml

```diff
@@ -3,9 +3,9 @@ name: agent-control
 description: Bootstraps New Relic' Agent Control
 
 type: application
-version: 0.0.68
+version: 0.0.69
```

Comment:
While working on the uninstallation and installation job I asked @paologallinaharbur to know if we could have a single PR updating both the `agent-control` and `agent-control-deployment` chart versions. I don't remember if he told me that "it's not possible" or that "he was not sure".

Do you have more knowledge on that? Maybe the tests fail because of that.

Reply:
We can try, there are two things that give me hope:
 - I believe that it was no supported by the tool we were using, but it was 3-4 years ago, therefore maybe now it works.

 - moreover, it is possible that since the way we manage child charts is different (via the job and not direct dependency) it could just work without publishing first the child chart

Reply:
Now for sure it is supported https://github.com/helm/chart-releaser/issues/411

Reply:
Done

Reply:
if we want we could disable the healthcheck during the installation